src/main/java/uk/gov/hmcts/reform/prl/services/caseinitiation/CaseInitiationService.java

```diff
@@ -46,4 +54,20 @@ public void handleCaseInitiation(String authorisation, CallbackRequest callbackR
 
         eventPublisher.publishEvent(new CaseDataChanged(caseData));
     }
+
+    public Map<String, Object> prePopulateCourtDetails(String authorisation, Map<String, Object> caseDataUpdated) {
```

Comment:
Would it be better to change this method name to prePopulateCourtDetailsList? I've noticed that there is already a method called prePopulateCourtDetailsList in CallbackController.java - at line 279.

Reply:
Hi @priteshkr please suggest if I should make the changes based on review comments since the ticket is already PO approved..